src/components/TestSuites.tsx

```diff
@@ -312,6 +260,20 @@ const TestCase: React.FC<TestCaseProps> = (props) => {
                 <Flex>
                     <FlexItem>{renderStatusIcon(test.status)}</FlexItem>
                     <FlexItem className={nameClassName}>{test.name}</FlexItem>
+                    {machineArchitecture && (
```

Comment:
Can we please create simple component, that takes param and returns `null` or payload. `{test && ` is OK, but this makes code less readable.

Reply:
or arch, and move `const machineArchitecture = getProperty(test, 'baseosci.arch');
` to component.

Reply:
1. Do you know if we can move `.test.` files under common directory? To not mix-up code and tests in the same dir
2. Please put a link to Jira-ticket to in git commit header or message

Reply:
I've moved the label to a separate component.

Yes, tests can be placed anywhere as long as they have the `.test.*` suffix. I actually slightly prefer having unit tests close to the code they're testing since there's no ambiguity in which file/module they belong to (if there are multiple files with the same name, for instance). Let me know if you have a strong preference on this. I can do the move in a separate PR.

I mention the Jira ticket in the corresponding commit message and in the PR description.